pkg/controllers/user/logging/utils/syslog.go

```diff
@@ -113,3 +113,23 @@ func getSeverity(severityStr string) syslog.Priority {
 
 	return syslog.LOG_INFO
 }
+
+func getWrapSeverity(severity string) string {
+	severityMap := map[string]string{
```

Comment:
I would prefer only mapping `warning` to `warn` and return the `severity` if we can not find a match in the map.

Reply:
add a map here in case we need to adapt other values, and use the "notice" as default, in case we get nothing.

Reply:
I don't like setting default value here. I would like to return `severity` itself so we can know what's wrong if someone mess this up.